spec/neo4j/driver/util/server_version_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe Neo4j::Driver::Internal::Util::ServerVersion do
+  context 'when', RUBY_PLATFORM: 'ruby' do
```

Comment:
have you verified that this actually works and the specs do not run on jruby?

Reply:
I have tested it against java and working fine but will verify again and update it here.

Reply:
Working should mean the spec is skipped on jruby.